pkg/sql/parser/sql.y

```diff
@@ -2972,11 +2972,27 @@ backup_options:
   }
 | REVISION_HISTORY
   {
-    $$.val = &tree.BackupOptions{CaptureRevisionHistory: true}
+    $$.val = &tree.BackupOptions{CaptureRevisionHistory: tree.MakeDBool(true)}
+  }
+| REVISION_HISTORY '=' TRUE
```

Comment:
yep that would be best

Reply:
done, this look good?

Reply:
yep, however now that it is an expr, you'll need to evaluate it rather than just compare it to DBoolTrue below, and I'm now looking at PlanHook and I'm not actually sure if we have that eval + check plumbed to here; might need to ask some sql folks how we should be eval'ing and typing as bool

Reply:
I think since you know that this expression must be of type bool, you can 
type check it yourself.
```
typedExpr, err := expr.TypeCheck(ctx, semaCtx, types.Bool)
```